### PR TITLE
feat: remove need for global wasm-bindgen-cli install

### LINKED ISSUE #4: Ensure wasm-bindgen-cli version matches dependency

### LINKED ISSUE #6: Don't require installing wasm-bindgen-cli globally
Global commands are a huge headache because version differences cause build failures (wasm-bindgen-cli gives a very unhelpful message when this occurs). It would be nice if this didn't require having to install wasm-bindgen-cli globally.

### PR DESCRIPTION
**Pull request by dsherret**
Uses wasm-bindgen-cli in Wasm in order to remove the need for a gobal wasm-bindgen-cli dependency.

Closes #6 
Closes #4

### CODE REVIEW

File: main.ts
Comment (dsherret) [inline main.ts:77]: Blocked on https://github.com/rustwasm/wasm-bindgen/pull/2913

File: deno.json
Comment (littledivy) [inline deno.json:12]: Since we are dogfooding, Can we just use the local version? Sounds like a good 'test'

**Review by littledivy — APPROVED** [review]
LGTM

File: deno.json
Comment (dsherret) [reply deno.json:12]: We can't really depend on the local version because if we break it then we can no longer build to fix it 😄

I'm going to update this to use this version once we merge it to main.

**Pull request merged**
